hugo/content/030-architecture/30_kubify/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: kubify
+remote: https://github.com/gardener/kubify.git
```

Comment:
section titles are rendered with a very small font for this remote page

Reply:
the reason for this is, that the page uses "####" as markdown. This is indeed very small.
The solution is to fix the remote document and use correct markdown headers like "#" or "##"